docs/src/main/sphinx/connector/hive.rst

```diff
@@ -623,6 +623,8 @@ Property Name                              Description
        ``KERBEROS``. Default is ``NONE``.
    * - ``hive.metastore.thrift.impersonation.enabled``
      - Enable Hive metastore end user impersonation.
+   * - ``hive.metastore.thrift.use-spark-table-statistics``
+     - Enable usage of table statistics generated by Apache Spark when hive table statistics are not available
```

Comment:
This property name gives the impression that the spark table stats are preffered to Trino table stats.
Previously we had a property name containing `fallback` in it.

Reply:
I've followed suggestion from @raunaqmorarka about property name. Could we agree on one name?